.travis.yml

```diff
@@ -13,15 +17,23 @@ env:
 
 r_packages:
   - devtools
+  - Cairo # this was missing on OSX, no idea why this wasn't installed :-( Needed for repr
 
 r_github_packages:
   # we currently still need IRdisplay from github in all cases...
   - IRkernel/IRdisplay
 
 
 script:
-  - source .travis-build.sh
+  - export CHECK_LOG=${TRAVIS_BUILD_DIR}/IRkernel.Rcheck/00check.log
+  - export INSTALL_LOG=${TRAVIS_BUILD_DIR}/IRkernel.Rcheck/00install.out
+  - bash .travis-build.sh
```

Comment:
maybe the sourcing was necessary to access the travis_fold function. if so, please revert this line